src/Features/Core/Portable/PreferFrameworkType/PreferFrameworkTypeDiagnosticAnalyzerBase.cs

```diff
@@ -4,12 +4,11 @@
 
 #nullable enable
 
-using System;
 using System.Collections.Immutable;
-using System.Threading;
 using Microsoft.CodeAnalysis.CodeStyle;
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Options;
+using OptionSet = Microsoft.CodeAnalysis.Options.OptionSet;
```

Comment:
Is this one needed considering the line above?

Reply:
Hmm, I will clean this - it should not be needed.